include/aws/http/request_response.h

```diff
@@ -515,14 +586,35 @@ struct aws_http_message *aws_http_message_new_request_with_headers(
     struct aws_http_headers *existing_headers);
 
 /**
- * Create a new response message.
+ * Create a new response message. HTTP/1.1 response
  * The message is blank, all properties (status, headers, etc) must be set individually.
```

Comment:
I think, for backwards compatibility, we're going to need to support old-style messages into HTTP/2. We'll just need to run them through a "transformation"

add comments explaining this